.github/workflows/ci.yaml

```diff
@@ -114,10 +114,11 @@ jobs:
         path:
         - .
         - ./tests/integration/relations/opensearch_provider/application-charm/
-    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v22.0.0
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@beta-charmcraftst124
```

Comment:
Maybe add some `# todo` comments here?

Reply:
The big "TODO" is on `charmcraft.yaml` explaining why all of this is needed. Once we generally update how charmcraft is done, all of this other places will be forced to change by necessity.